i18n/en.json

```diff
@@ -29,5 +29,10 @@
 		"report-bug": "Report a bug",
 		"revision-edit-summary-removed": "edit summary removed",
 		"view-source-code-on": "View on $1",
-		"warning-no-results": "No Results"
+		"warning-no-results": "No Results",
+		"discuss-on-wiki": "Share these results",
+		"discuss-on-wiki-text": "Between $1 and $2 on $3 [[User:$4]] ([[User talk:$4|talk]] | ([[Special:Contributions/$4|contribs]]) and [[User:$5]] ([[User talk:$5|talk]] | ([[Special:Contributions/$5|contribs]]) both made edits to some pages in common. You can see a chronological list of all their edits on the [$6 Interaction Timeline] or a table view on the [$7 Interaction Analyser].",
```

Comment:
`Between 2000-01-01 and 2018-08-02 on en.wikipedia.org User:DMaza (talk | (contribs) and ...`
I think there is an unnecessary  `(` before the contribs link